crates/storage/src/column.rs

```diff
@@ -63,6 +63,9 @@ pub enum Column {
     // TODO: Remove this column and use `Metadata` column instead.
     /// Table for genesis state import progress tracking.
     GenesisMetadata = 20,
+
+    /// See [`Blobs`](fuel_vm_private::storage::BlobData)
+    Blobs = 21,
```

Comment:
You can use number `20` and move `GenesisMetadata` to the `0x21`. It is not used anywhere except the regenesis

Reply:
baae7f513